Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -977,9 +974,9 @@ private static void runQuickTests(MinioClient minioClient)
                     await minio.CopyObjectAsync(bucketName, objectName, destBucketName, destObjectName, conditions);
 
                 }
-                catch (MinioException)
+                catch (MinioException ex)
                 {
-                    Assert.Fail();
+                    throw ex;
```

Comment:
This is more like a generic approach question/comment:
If we `catch` and simply `throw` the caught exception without adding our customized, more explanatory error message, then what is the point of catching these exceptions?
Wouldn't the existing code and approach be the same with not doing anything and letting the system built-in exception mechanism do its job and throw these exceptions on its own without our `catch` logic get in the way?

Reply:
our objective here is to capture the exception in mint logger, which will be served by cascading error up the stack to calling function - the assert statement just causes mint to crash with a generic aggregate exception.